src/_guides/language/analysis-options.md

```diff
@@ -92,15 +92,10 @@ add static analysis to your tool, see the
 Place the analysis options file, `analysis_options.yaml`,
 at the root of the package, in the same directory as the pubspec file.
 
-{{site.alert.warn}}
-  **Breaking change:** The conventional name for the analysis options file
+{{site.alert.note}}
+  The conventional name for the analysis options file
   used to be `.analysis_options` (note the leading dot and missing `.yaml` suffix).
-  We expect support for the `.analysis_options` name to go away in a future
-  release, so we recommend that you **rename your `.analysis_options` files to
-  `analysis_options.yaml`.**
-  {% comment %}
-  Tracking issue: https://github.com/dart-lang/sdk/issues/28385
-  {% endcomment %}
+  Support for the `.analysis_options` name was dropped in Dart 2.8.
```

Comment:
"The older name for the analysis options file was '.analysis_options'. Support for that name for the file was dropped in Dart 2.8. To upgrade, simply rename the file to 'analysis_options.yaml' - the formats are the same" ?

Or similar - this just retains the advise about how to update if you're using the older file name.